service/src/main/java/org/databiosphere/workspacedataservice/dao/RecordDao.java

```diff
@@ -1024,4 +1024,14 @@ public void deleteRecordType(UUID instanceId, RecordType recordType) {
       throw e;
     }
   }
+
+  public void deleteAttribute(UUID instanceId, RecordType recordType, String attribute) {
+    namedTemplate
+        .getJdbcTemplate()
+        .update(
```

Comment:
Can you use the varargs update here?  I am guessing it's safer than doing string interpolation:
```suggestion
        .update("alter table ? drop column ?", getQualifiedTableName(recordType, instanceId), quote(SqlUtils.validateSqlString(attribute, ATTRIBUTE)))
```